sunkit_image/tests/conftest.py

```diff
@@ -173,3 +174,37 @@ def aia_171(request):
     if request.param == "map":
         return smap
     return smap.data if request.param == "array" else None
+
+
+@pytest.fixture()
+def hmi_map():
+    # Creating the data
+    np.random.default_rng(42)
+    data = np.random.default_rng(42).random((50, 50))
+    # Create a circular feature for representing the sun
+    theta = np.linspace(0, 2 * np.pi, 200)
+    circle_x = 25 + 20 * np.cos(theta)
+    circle_y = 25 + 20 * np.sin(theta)
+
+    for x, y in zip(circle_x, circle_y, strict=False):
+        data[int(y), int(x)] = -50
+    # Smooth the data
+    data = gaussian_filter(data, sigma=2)
+
+    # Create the header with appropriate metadata
+    header = {
+        "date-obs": "2022-01-01T00:00:00.000",
+        "crpix1": 50,
+        "crpix2": 50,
+        "cdelt1": 1,
+        "cdelt2": 1,
+        "crval1": 0,
+        "crval2": 0,
+        "cunit1": "arcsec",
+        "cunit2": "arcsec",
+        "ctype1": "HPLN-TAN",
+        "ctype2": "HPLT-TAN",
+        "rsun_obs": 2500,
+    }
+
+    return sunpy.map.GenericMap(data, header)
```

Comment:
What does the resulting map look like?

Reply:
![image](https://github.com/sunpy/sunkit-image/assets/117574289/2826c45b-b73b-4867-80c7-7833517bd57d)

I tried adding sunspots like the one below: 
![image](https://github.com/sunpy/sunkit-image/assets/117574289/0714d24d-b1b8-4e47-b9d3-ba350ba2f236)

but the stara isn't able to find the locations of those. While it identifies correctly for the real hmi continuum.

Reply:
So I wanted to ask, what should I focus on? creating the appropriate mock hmi map that can be detected by stara or just download a real hmi map and store it in a temporary directory and check? The second approach might fail if ever the fido client faces any issues in the future

Reply:
I guess the answer to the question is, what do you want to test with this?

Ideally, we can just take a real HMI image, downsample it and add as actual test data.
Or we can improve the test image creation here.

Reply:
We want to try and avoid using any test data in any gallery example.

Reply:
I think there is some hmi test data in the sunpy repo, i forgot if its LOS tho. That might be worth trying.

Reply:
> I think there is some hmi test data in the sunpy repo, i forgot if its LOS tho. That might be worth trying.

I think you mean this one, `2023_01_31__03_39_23_200__SDO_HMI_HMI_continuum.jp2`? Let me check if this works with stara.

Reply:
This is the one present in test folder. Tried up scaling too but still no luck in finding the regions.
![image](https://github.com/sunpy/sunkit-image/assets/117574289/d120ff9f-1f26-4a89-8e64-4bffdeeab5ba)

Reply:
The WCS on that image is busted. I wonder why.

So if that won't work, then we need to consider a better fake image. Can you create a 1k by 1k image with a "large sunspot" in the middle?